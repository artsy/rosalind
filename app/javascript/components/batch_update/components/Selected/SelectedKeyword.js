import React from 'react'
import SelectedComponent from './SelectedComponent'

const SelectedKeyword = props => {
  return <SelectedComponent name={props.text} onRemove={props.onRemove} />
}

export { SelectedKeyword }
