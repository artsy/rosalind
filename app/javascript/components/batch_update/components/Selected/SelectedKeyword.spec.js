import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'
import { SelectedKeyword } from './SelectedKeyword'

let props

beforeEach(() => {
  props = {
    text: 'soup',
    onRemove: jest.fn(),
  }
})

it('renders correctly', () => {
  const rendered = renderer.create(<SelectedKeyword {...props} />)
  const tree = rendered.toJSON()
  expect(tree).toMatchSnapshot()
})
