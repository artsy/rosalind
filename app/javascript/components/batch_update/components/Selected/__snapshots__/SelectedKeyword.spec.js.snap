// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c0 {
  font-weight: bold;
  margin: 0.25em 0;
}

.c0 a.remove {
  display: inline-block;
  height: 2em;
  width: 2em;
  line-height: 2em;
  text-align: center;
  color: #ddd;
  margin-left: 0.5em;
  border-radius: 1em;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c0 a.remove:hover {
  color: #333;
  cursor: pointer;
  background: #eee;
}

<div
  className="c0"
>
  soup
  <a
    className="remove"
    href="#"
    onClick={[Function]}
  >
    âœ•
  </a>
</div>
`;
